Jhia 7 - adding authentication flows to API (#2)
JumpTwentyFour/wandersnap-api: README.md

# Wandersnap Api

## Requirements

- Docker (https://docs.docker.com/get-docker/)
- NVM (https://github.com/nvm-sh/nvm#installing-and-updating)

## Setup

- `cp .env.example .env`
- `docker compose build`
- `docker compose up -d`
- `docker compose exec php composer install`
- `docker compose exec php php artisan key:generate`
- `docker compose exec php php artisan migrate`

> The app should now be able to hit a ping enpoint att (http://localhost:8000)

Reviewer: ```suggestion
The API should now be able available at (http://localhost:8000)
```
